### PR TITLE
chore(zh-cn): `update src/content/docs/develop/Debug/index.mdx`

### PR DESCRIPTION
**Pull request by RSS1102**
The `update src/content/docs/develop/Debug/index.mdx` file translate into Chinese

### CODE REVIEW

File: src/content/docs/zh-cn/develop/Debug/index.mdx [hunk 1]
```diff
@@ -1,15 +1,148 @@
...
+sidebar:
+  label: 概述
+  order: 10
+i18nReady: true
 ---
 
-import { LinkCard, CardGrid } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
 
-<CardGrid>
```
Comment (RSS1102) [inline src/content/docs/zh-cn/develop/Debug/index.mdx:8]: In order to ensure consistency with the original document format, I have deleted it here.

File: src/content/docs/zh-cn/develop/Debug/index.mdx [hunk 2]
```diff
@@ -1,15 +1,148 @@
 ---
 title: 调试
-description: 调试工作流程的技巧和窍门
+sidebar:
+  label: 概述
+  order: 10
+i18nReady: true
```
Comment (RSS1102) [inline src/content/docs/zh-cn/develop/Debug/index.mdx:6]: In order to ensure consistency with the original document format, I added this, but I don't know its usefulness or whether it is necessary.

**Comment by RSS1102** [comment]
#2494 It seems that it is no longer being updated. I tried to synchronize its code to the latest version and continue submitting, but was unsuccessful. I don't know if it was an error in the git command

**Comment by RSS1102** [comment]
https://github.com/tauri-apps/tauri-docs/pull/2494 It seems that it is no longer being updated. I originally planned to fork his code to continue making changes, but it seems that the same upstream can only be forked once. So it can only be cited and explained here.
（Because I have PRs provided by other branches, the current fork cannot be deleted.）
image

File: src/content/docs/zh-cn/develop/Debug/index.mdx (hunk 2 above)
Comment (vasfvitor) [reply src/content/docs/zh-cn/develop/Debug/index.mdx:6]: it's necessary, I think `i18nReady: true` is only necessary on root locale, but you can leave as is

File: src/content/docs/zh-cn/develop/Debug/index.mdx (hunk 1 above)
Comment (vasfvitor) [reply src/content/docs/zh-cn/develop/Debug/index.mdx:8]: it's ok, that's how it should be done. Always base on the original

**Review by vasfvitor — COMMENTED** [review]
not a review.
will try to review in a few days with translate if nobody who speaks Chinese does so

**Comment by RSS1102** [comment]
> not a review. will try to review in a few days with translate if nobody who speaks Chinese does so

ok， Thank you for your guidance. I hope everything goes smoothly and I will be able to translate faster in the future.😆😆

File: src/content/docs/zh-cn/develop/Debug/index.mdx [hunk 3]
```diff
@@ -1,15 +1,148 @@
...
+[dependencies]
+tauri = { version = "...", features = ["...", "devtools"] }
+```
+
+## 调试核心进程
+
+Core 进程由 Rust 提供支持，因此您可以使用 GDB 或 LLDB 对其进行调试。您可以按照 [Debugging in VS Code] 指南了解如何使用 LLDB VS Code 扩展来调试 Tauri 应用程序的核心进程。
+
+[debugging in vs code]: /develop/debug/vscode/
+[`WebviewWindow::open_devtools`]: https://docs.rs/tauri/2.0.0/tauri/webview/struct.WebviewWindow.html#method.open_devtools
```
Comment (vasfvitor) [inline src/content/docs/zh-cn/develop/Debug/index.mdx]: any reason as why didn't you translated the link to "Debugging in VS Code" and it's reference?

**Review by vasfvitor — APPROVED** [review]

File: src/content/docs/develop/Debug/index.mdx [hunk 4]
```diff
@@ -142,8 +142,8 @@ tauri = { version = "...", features = ["...", "devtools"] }
 
 ## Debugging the Core Process
 
-The Core process is powered by Rust so you can use GDB or LLDB to debug it. You can follow the [Debugging in VS Code] guide to learn how to use the LLDB VS Code Extension to debug the Core Process of Tauri applications.
+The Core process is powered by Rust so you can use GDB or LLDB to debug it. You can follow the [在 VS Code 中调试] guide to learn how to use the LLDB VS Code Extension to debug the Core Process of Tauri applications.
 
-[debugging in vs code]: /develop/debug/vscode/
+[在 VS Code 中调试]: /zh-cn/develop/debug/vscode/
```
Comment (vasfvitor) [inline src/content/docs/develop/Debug/index.mdx]: you did on the original file

Comment (RSS1102) [reply src/content/docs/develop/Debug/index.mdx]: sorry...

Comment (RSS1102) [reply src/content/docs/develop/Debug/index.mdx]: 😩 done ,so sorry

**Pull request merged**